control.go

```diff
@@ -132,7 +130,7 @@ func (c *Control) ListHostmapIndexes(pendingMap bool) []ControlHostInfo {
 }
 
 // GetHostInfoByVpnIp returns a single tunnels hostInfo, or nil if not found
-func (c *Control) GetHostInfoByVpnIp(vpnIp iputil.VpnIp, pending bool) *ControlHostInfo {
+func (c *Control) GetHostInfoByVpnIp(vpnIp netip.Addr, pending bool) *ControlHostInfo {
```

Comment:
Should we unmap for the caller here or expect they have unmapped the addr?

Reply:
as best as I can tell, this is only called in test code, so I think we can err on the side of calling `Unmap` internally here.